qiskit_experiments/hamiltonian_tomography/cr_hamiltonian_experiment.py

```diff
@@ -0,0 +1,299 @@
+            try:
+                self.set_experiment_options(dt=backend.configuration().dt)
+            except AttributeError as ex:
+                raise QiskitError(
+                    "Duration is not set and the backend doesn't provide dt. "
+                    "Set time resolution of the system to proceed."
+                ) from ex
+
+        durations_dt = np.asarray(
+            [self._to_samples(dur) for dur in self.experiment_options.durations], dtype=int
+        )
+
+        if not self._cr_gate_schedule:
+            # defaults to the CR1 sequence
+            par_duration = circuit.Parameter("duration")
+            amp = self.experiment_options.amp
+            sigma_dt = self._to_samples(self.experiment_options.sigma, require_integer=False)
+            risefall = self.experiment_options.risefall
+
+            with pulse.build(backend=backend) as cr_sched:
+                with pulse.align_left():
+                    pulse.play(
+                        pulse.GaussianSquare(
+                            duration=par_duration,
+                            amp=amp,
+                            sigma=sigma_dt,
+                            width=par_duration - 2 * risefall * sigma_dt,
+                        ),
+                        pulse.control_channels(*self.physical_qubits)[0],
+                    )
+                    pulse.delay(par_duration, pulse.drive_channel(self.physical_qubits[0]))
+                    pulse.delay(par_duration, pulse.drive_channel(self.physical_qubits[1]))
+            cr_sched.metadata = {
+                "amplitude": amp,
+                "sigma": sigma_dt,
+                "risefall": risefall,
+            }
+
+            if self._x_values is None:
+                flat_top_width_dt = durations_dt - 2 * risefall * sigma_dt
+                net_risefall_dt = np.sqrt(2 * np.pi * sigma_dt ** 2)
+                x_values = (flat_top_width_dt + net_risefall_dt) * self.experiment_options.dt
+            else:
+                x_values = self._x_values
+        else:
+            # user defined CR sequence
+            cr_sched = self._cr_gate_schedule
+
+            if self._x_values is None:
+                x_values = durations_dt * self.experiment_options.dt
+            else:
+                x_values = self._x_values
+
+        par_durations = list(cr_sched.get_parameters("duration"))
+
+        # create tomography circuits
+        cr_gate = circuit.Gate(name="cr_gate", num_qubits=2, params=par_durations)
+        temp_circs = list()
+        for control_state in (0, 1):
+            for meas_basis in ("x", "y", "z"):
+                tomo_circ = circuit.QuantumCircuit(2, 1)
+                if control_state:
+                    tomo_circ.x(0)
+                tomo_circ.append(cr_gate, [0, 1])
+                if meas_basis == "x":
+                    tomo_circ.h(1)
+                elif meas_basis == "y":
+                    tomo_circ.sdg(1)
+                    tomo_circ.h(1)
```

Comment:
Suppose we want to run this with an instance of calibrations. We need to attach pulse schedules to these post rotations. Can you instead use `circ.rx(np.pi/2, 1)` and `circ.ry(np.pi/2, 1)`? Like that the transpiler can handle them if we do not specify schedules for them but it also gives us the means to attach custom schedules from the cals.